crates/ironrdp-cliprdr/src/lib.rs

```diff
@@ -31,15 +30,23 @@ use crate::pdu::FormatList;
 /// PDUs for sending to the server on the CLIPRDR channel.
 pub type CliprdrSvcMessages<R> = SvcProcessorMessages<Cliprdr<R>>;
 
-#[derive(Debug, Error)]
+#[derive(Debug)]
 enum ClipboardError {
-    #[error("received clipboard PDU is not implemented")]
     UnimplementedPdu { pdu: &'static str },
-
-    #[error("sent format list was rejected")]
     FormatListRejected,
 }
 
+impl core::fmt::Display for ClipboardError {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        match self {
+            ClipboardError::UnimplementedPdu { pdu } => {
+                write!(f, "received clipboard PDU `{pdu}` is not implemented")
+            }
+            ClipboardError::FormatListRejected => write!(f, "Sent format list was rejected"),
```

Comment:
```suggestion
            ClipboardError::FormatListRejected => write!(f, "sent format list was rejected"),
```

https://github.com/Devolutions/IronRDP/blob/7a0fcce203e5f8ca522582b860ba9347c1256be8/STYLE.md#formatting-of-error-messages